### DISCUSSION TITLE
question about selfhosted relay and id servers

### DISCUSSION THREAD

**Discussion by SkySkyInTheSky** [rustdesk/rustdesk#5460] [Q&A]
Hello. I read docs, but i have a question.

How do I configure it as below?

A is connected by B/D.
B is connected to C.
A/B/D uses the Internet (external network), and C does not. (Internal Network)

     [ D - client ]
       |
     [ A - server ]  ---  [ B - relay ]   ---  [ C - client ]
                                 
In this case, how do I set it up to connect directly from D to C?

I bought a business plan.
Even if I put IP in RELAY in the settings, I can't access it.

Thanks and best regards.

**Comment by rustdesk** [comment]
@dinger1986

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet

**Comment by dinger1986** [comment]
As long as your pointing to your internal server and it’s the same as the external server it’ll work fine, or look at pin hole nat and sorting that, what router do you have?

**Reply by SkySkyInTheSky** [reply]
Sorry, I didn't understand your question.

**Reply by dinger1986** [reply]
I don’t understand if the server is on the lan and the clients are connected to it either remotely or onsite it should work fine. Why can’t you use your external IP for both?

**Reply by SkySkyInTheSky** [reply]
A and D are in the same place.
B and C are in the same place.

The rust server is in A.
D is a personal PC.

B can access C.
C consists of an intranet for security reasons.

I want to connect directly from D to C through B connected to A.

In this case, how do I set it up?

**Comment by SergeyMy** [comment]
[ C - client ] must see [ A - server ]
